### PR TITLE
all: goimports -w .

### PR DESCRIPTION
**Pull request by sashamelentyev**
Signed-off-by: Sasha Melentyev <sasha@melentyev.io>

### CODE REVIEW

File: tests/integration/clientv3/connectivity/dial_test.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 	"go.etcd.io/etcd/client/v3"
 	integration2 "go.etcd.io/etcd/tests/v3/framework/integration"
 	"go.etcd.io/etcd/tests/v3/framework/testutils"
-	clientv3test "go.etcd.io/etcd/tests/v3/integration/clientv3"
+	"go.etcd.io/etcd/tests/v3/integration/clientv3"
```
Comment (ahrtr) [inline tests/integration/clientv3/connectivity/dial_test.go:29]: It's updated by `goimports -w` automatically? Why did previous file add `clientv3test`, while this file remove `clientv3test`?

Comment (sashamelentyev) [reply tests/integration/clientv3/connectivity/dial_test.go:29]: goimports -w add this:
```go
	pb "go.etcd.io/etcd/api/v3/etcdserverpb"
	"go.etcd.io/etcd/client/pkg/v3/transport"
	integration2 "go.etcd.io/etcd/tests/v3/framework/integration"
	"go.etcd.io/etcd/tests/v3/framework/testutils"
	"go.etcd.io/etcd/tests/v3/integration/clientv3"
	clientv3test "go.etcd.io/etcd/tests/v3/integration/clientv3"
	"google.golang.org/grpc"
```

Mb I remove changes in tests?

Comment (ptabor) [reply tests/integration/clientv3/connectivity/dial_test.go:29]: We want to keep 

```
clientv3test "go.etcd.io/etcd/tests/v3/integration/clientv3"
```

And change in this file goes in the other direction. So it's why we are surprised.

Comment (sashamelentyev) [reply tests/integration/clientv3/connectivity/dial_test.go]: Fixed.

I add `clientv3test "go.etcd.io/etcd/tests/v3/integration/clientv3"`myself and run goimports and all ok. Rerun goimports have no changes

**Comment by ptabor** [comment]
Please add it to: 
https://github.com/etcd-io/etcd/blob/main/scripts/fix.sh

as this change makes sense if we want to keep it consistent going forward. 

And the benefit of this change is that there is less 'surprise' that the `/client/v3` package is not addressable by `v3`.

**Comment by sashamelentyev** [comment]
> Please add it to: https://github.com/etcd-io/etcd/blob/main/scripts/fix.sh
> 
> as this change makes sense if we want to keep it consistent going forward.
> 
> And the benefit of this change is that there is less 'surprise' that the `/client/v3` package is not addressable by `v3`.

After `goimports -w .` have changes in *.pb.go files...

**Review by ahrtr — APPROVED** [review]
LGTM

Thank you @sashamelentyev 

Please consider to resolve https://github.com/etcd-io/etcd/pull/14788#issuecomment-1318883581 suggested by @ptabor , of course in a separate PR.

**Pull request merged**